.gitignore

```diff
@@ -1,56 +1,32 @@
-
-# Created by https://www.toptal.com/developers/gitignore/api/go,rust
-# Edit at https://www.toptal.com/developers/gitignore?templates=go,rust
-
-### Go ###
-# Binaries for programs and plugins
+# Binary suffixes
 *.exe
 *.exe~
 *.dll
 *.so
 *.dylib
 
-# Monero dir
+# Downloaded monero binaries
 /monero.tar.bz2
 /monero-*
 
-# bin
+# Generated binaries
 swapcli
 swapd
 swaprecover
 
 # Test binary, built with `go test -c`
 *.test
 
-# Output of the go coverage tool, specifically when used with LiteIDE
-*.out
+# Output of the go coverage tool
+/coverage.out
 /coverage.html
 
-# Dependency directories (remove the comment below to include it)
-# vendor/
-
-### Go Patch ###
-/vendor/
-/Godeps/
```

Comment:
I didn't like having all the boilerplate. If a `vendor` folder showed up in the project, it is something we'd want to know about.  There are no rust artifacts in the top-level git repo, so I cleared them out.